### PR TITLE
[OSDOCS#11226] Release notes for ShiftStack 4.17

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.17

Issue: [OSDOCS-11226](https://issues.redhat.com//browse/OSDOCS-11226)

Link to docs preview: https://82042--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html

QE review:
- [x] QE has approved this change.

Additional information: Bug fixes are handled outside of this PR. I'll squash before merge.

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -742,12 +749,12 @@ In the following tables, features are marked with the following statuses:
 |`compute.platform.openstack.rootVolume.type` for {rh-openstack}
 |Deprecated
 |Deprecated
-|Deprecated
+|Removed
```
Comment (maxwelldb) [inline release_notes/ocp-4-17-release-notes.adoc:752]: Confirm removal vs. continuing deprecation

Comment (maxwelldb) [reply release_notes/ocp-4-17-release-notes.adoc:752]: @pierreprinetti Would you be able to comment on this and the other deprecated item change?

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -59,6 +59,21 @@ This release adds improvements related to the following components and concepts:
...
+For more information, see xref:../installing/installing_openstack/installing-openstack-three-node.adoc#installing-openstack-three-node#installing-openstack-three-node[Installing a three-node cluster on OpenStack].
+
+[id="ocp-4-17-deploying-osp-with-root-volume-etcd-on-local-disk"]
+==== Deploying {rh-openstack-first} with root volume and etcd on local disk (Generally Available)
+
+You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local disk as a Day 2 deployment with this generally available feature.
+
+For more information, see xref:../installing/installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc#deploying-openstack-with-rootvolume-etcd-on-local-disk[Deploying on OpenStack with rootVolume and etcd on local disk].
+
+
```
Comment (maxwelldb) [inline release_notes/ocp-4-17-release-notes.adoc:76]:
```suggestion
```

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (mandre) [reply release_notes/ocp-4-17-release-notes.adoc:764]: We haven't removed it, so still Deprecated.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -742,12 +749,12 @@ In the following tables, features are marked with the following statuses:
...
 |Deprecated
 |Deprecated
-|Deprecated
+|Removed
 
 |`controlPlane.platform.openstack.rootVolume.type` for {rh-openstack}
 |Deprecated
 |Deprecated
-|Deprecated
+|Removed
```
[the remark this answers is not in the document]
Comment (mandre) [reply release_notes/ocp-4-17-release-notes.adoc:769]: Same here, it's still there.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 4]
```diff
@@ -747,12 +761,12 @@ In the following tables, features are marked with the following statuses:
...
```
Comment (maxwelldb) [inline release_notes/ocp-4-17-release-notes.adoc:769]:
```suggestion
|Deprecated
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 5]
```diff
@@ -85,6 +85,20 @@ For more information on required permissions, see xref:../installing/installing_
 
 With this release, you can install {product-title} on {aws-first} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional {aws-short} configuration parameters].
 
+[id="installation-and-update-shiftstack-compact_{context}"]
+==== Three-node cluster support for {rh-openstack]
```
Comment (rlobillo) [inline release_notes/ocp-4-17-release-notes.adoc:89]:
```suggestion
==== Three-node cluster support for {rh-openstack}
```

Comment (maxwelldb) [reply release_notes/ocp-4-17-release-notes.adoc:89]: Well played :upside_down_face:.

**Comment by eurijon** [comment]
Hey, regarding the compact clusters support, the epic in 4.17 is targeting the IPI compact clusters (and not UPI).
We added the automation and jobs for IPI compact only, and would need to do some additional work for UPI compact. I'm ok working on it but for now the support should be only for IPI compact clusters.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 6]
```diff
@@ -85,6 +85,20 @@ For more information on required permissions, see xref:../installing/installing_
 
 With this release, you can install {product-title} on {aws-first} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional {aws-short} configuration parameters].
 
+[id="installation-and-update-shiftstack-compact_{context}"]
+==== Three-node cluster support for {rh-openstack}
+
+Deploying a three-node cluster is now supported on {rh-openstack-first}.
```
Comment (maxwelldb) [inline release_notes/ocp-4-17-release-notes.adoc:91]:
```suggestion
Deploying a three-node cluster on installer-provisioned infrastructure is now supported on {rh-openstack-first}.
```

**Comment by eurijon** [comment]
@maxwelldb it looks good now from compact support perspective. I want to add that we should remove the UPI compact support from https://github.com/openshift/openshift-docs/pull/80595 as well (sorry, didn't notice before merged).
Once we add UPI compact coverage we'll let you know so we can add it back in the documentation.

**Comment by maxwelldb** [comment]
> @maxwelldb it looks good now from compact support perspective. I want to add that we should remove the UPI compact support from #80595 as well (sorry, didn't notice before merged). Once we add UPI compact coverage we'll let you know so we can add it back in the documentation.

Sure. Requesting an ack on #82453 after the preview posts. @eurijon

**Comment by eurijon** [comment]
lgtm

**Review by sheriff-rh — COMMENTED** [review]
LGTM! Don't forget to squash commits! :)

**Comment by maxwelldb** [comment]
Squashed

**Pull request merged**